src/createServer.js

```diff
@@ -1,3 +1,70 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const SUPPORTED_CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const [url, params] = req.url.split('?');
+    const quertParams = new URLSearchParams(params);
+    const targetCase = quertParams.get('toCase');
+    const originalText = url.slice(1);
+
+    if (
+      originalText.length === 0
+      || !SUPPORTED_CASES.includes(targetCase)
+      || !targetCase
+    ) {
```

Comment:
you have other checks below
```suggestion
    if (originalText.length === 0) {
```